addons/website/static/src/js/widgets/dialog.js

```diff
@@ -0,0 +1,10 @@
+/** @odoo-module **/
+
+import Dialog from 'web.Dialog';
+
+Dialog.include({
+    init: function (parent, options) {
+        options.remainingModalSelector = '.modal[role="dialog"]:not([data-scroll-body="true"])';
```

Comment:
Should this be a method that return a set of elements in the web module, and which is overriden here to exclude stuff from it?

The current implementation doesn't seems modular, if another module do the same, only the `remainingModalSelector` from the last module will be considered, right?

Eg, here if someone later in `portal` do 
```
        options.remainingModalSelector = '.modal[role="dialog"]:not([whatever="true"])';
```
it will be wiped and ignored once website is installed

While exluding records from the initial element set from web would work, website would remove stuff from it (data-scroll-body), then portal would remove `data-whatever` from the set etc

Reply:
Introduced a `_getScrollableBodyExclusions()` to build a list of `:not(...)`